### ISSUE TITLE
Video infinitively loading

### ISSUE THREAD

**Issue by rgabbo** [TeamNewPipe/NewPipe#7540]
### Checklist

- [x] I am using the latest version - 0.21.14 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1. Go to https://www.youtube.com/watch?v=aP6yRL5pn_o

### Actual behavior

I get an infinite loading wheel

### Expected behavior

Video should play fine

### Device info

 - Android version/Custom ROM version: 10
 - Device model: cat S52

Tried also in an emulator with a fresh install but I get the same result

**Comment by litetex** [comment]
> 1. Go to https://www.youtube.com/watch?v=aP6yRL5pn_o

Uhm it works as expected on my phone:
![grafik](https://user-images.githubusercontent.com/40789489/146061990-48bd35d6-41f0-4ffa-a528-11992008e1f5.png)

Can you recheck please? Do you maybe have any special settings set?

**Comment by rgabbo** [comment]
I just checked and issue is not reproducible neither in my phone nor in the emulator (so with no special setting). Maybe a temporary server error.

**Comment by MathJoDE** [comment]
I also observed this obscure issue two times in the last week. 
Always with newly uploaded videos (<12 hours)

Closing the app, clearing cache and restarting did not help, for this specific video the video still infinitely loads. **Playing just the audio in the background works fine**.

However after some time (a day or so) the bug is gone.

Are there any logs in newpipe (standard app, newest version) one can look into?
